packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.8.1
+
+* Add a new method `getMultiImage` to allow picking multiple images on iOS 14 or higher and Android 4.3 or higher.
```

Comment:
Maybe nice to mention what this function does on older devices. Does it pick 1 or does it not work at all?